lib/exception_hunter/error_creator.rb

```diff
@@ -1,14 +1,18 @@
 module ExceptionHunter
   class ErrorCreator
+    HTTP_TAG = 'HTTP'.freeze
+    WORKER_TAG = 'Worker'.freeze
+    MANUAL_TAG = 'Manual'.freeze
+
     class << self
-      def call(**error_attrs)
+      def call(tag: HTTP_TAG, **error_attrs)
```

Comment:
Thinking about this again we might not want a default tag, we don't even use it. Anyway, not a blocker